test-utils/state-viewer/src/cli.rs

```diff
@@ -0,0 +1,244 @@
+use std::path::{Path, PathBuf};
+use std::sync::Arc;
+
+use clap::{AppSettings, Clap};
+use lazy_static::lazy_static;
+
+use near_logger_utils::init_integration_logger;
+use near_primitives::types::{BlockHeight, ShardId};
+use near_primitives::version::{DB_VERSION, PROTOCOL_VERSION};
+use near_store::{create_store, Store};
+use nearcore::{get_default_home, get_store_path, load_config, NearConfig};
+
+use crate::commands::*;
+
+lazy_static! {
+    static ref DEFAULT_HOME: PathBuf = get_default_home();
+}
```

Comment:
For newer code, I'd weakly recommend using `once_cell` over `lazy_static`, but that's not super important.